virttest/qemu_vm.py

```diff
@@ -2043,8 +2048,8 @@ def sort_key(dev):
                     "spice_x509_key_file", "spice_x509_cacert_file",
                     "spice_x509_key_password", "spice_x509_secure",
                     "spice_x509_cacert_subj", "spice_x509_server_subj",
-                    "spice_secure_channels", "spice_image_compression",
-                    "spice_jpeg_wan_compression",
+                    "spice_secure_channels", "spice_plaintext_channels",
```

Comment:
Okay, seems fine for me.
@rdkdd please update tp-spice/spice/lib/stest.py 

```
KVM_SPICE_KNOWN_PARAMS = [
    "spice_port",
    "spice_password",
...
```

When this pull req is merged.

ACK.